release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -976,6 +992,25 @@ If you are receiving pod security violations, see the following resources:
 
 * If necessary, you can set a custom admission profile on the namespace or pod by setting the `pod-security.kubernetes.io/enforce` label.
 
+[discrete]
+[id="ocp-4-12-olm-catalogs-and-psa-restricted-enforcement"]
+==== Catalog sources and restricted pod security admission enforcement
+
+Catalog sources built using the SQLite-based catalog format and a version of the `opm` CLI tool released before {product-title} 4.11 cannot run under restricted pod security enforcement.
+
+In {product-title} {product-version}, namespaces do not have restricted pod security enforcement by default and the default catalog source security mode is set to `legacy`.
+
+If you do not want to run your SQLite-based catalog source pods as restricted, you do not need to update your catalog source in {product-title} {product-version}. However, it is recommended that you take action now to ensure your catalog sources run under restricted pod security enforcement. If you do not take action to ensure your catalog sources run under restricted pod security enforcement, your catalog sources might not run in future {product-title} releases.
```

Comment:
ISG generally avoids saying something is recommended and instead just advises to tell users to just do something...not sure how to reword this completely, but what do you think about:
```suggestion
To run your SQL-based catalog source pods under restricted pod security enforcement, update your catalog source in {product-title} {product-version}. This ensures that your catalog sources run future {product-title} releases.